tools/accuracy_checker/accuracy_checker/representation/face_98_landmarks_representation.py

```diff
@@ -0,0 +1,41 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+from .base_representation import BaseRepresentation
+
+
+class Face98LandmarksRepresentation(BaseRepresentation):
```

Comment:
i think, the special case of this representation no in number of points, it has heatmap
```suggestion
class FaceLandmarksHeatMapRepresentation(BaseRepresentation):
```